### DISCUSSION TITLE
Recognizing Common Mods for In Silico libraries

### DISCUSSION THREAD

**Discussion by enonimos** [vdemichev/DiaNN#1255] [Q&A] [answered]
For spec lib formatted in silico libraries that are built using DIA-NN, if any common mods, such as M(ox) or phospho, were enabled when it was created (under precursor ion generation), do the same mods need to be "checked off" for future data analyses? Or does DIANN automatically recognize these common mods in the speclib, independent of the precursor ion generation settings?
Thanks,
Todd

**Comment by vdemichev** [comment, marked as answer]
Hi Todd,

The effects of declaring variable modifications are basically the following:
- during FASTA digest, this controls whether peptides with these modifications are being generated and added to the library
- during raw files analysis, this determines if the modifications will be localised, which in turn will affect quite a number of things, possibly even identiifcation numbers.

Best,
Vadim

**Reply by enonimos** [reply]
Hi Vadim, thank you for the information about the variable modifications. One quick follow-up on the second bullet point. With localization turned on, if the identification numbers did change, would you usually predict a loss in sensitivity, due to the additional "Peptidoform.Q.Value" type filter?

**Reply by vdemichev** [reply]
Normally this will be slight 'beneficial' loss, wherein incorrect peptidoform assignments will be filtered out.

Best,
Vadim
